Resources/Textures/_DV/Objects/Tools/Decoys/juggernaut_decoy.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
     "version": 1,
     "license": "CC-BY-SA-3.0",
-    "copyright": "Taken from Wizard's Den, helmet modified by @bogus_0451",
+    "copyright": "Taken from Resources/Textures/Objects/Tools/Decoys/juggernaut_decoy.rsi, helmet modified by @bogus_0451",
```

Comment:
use an actual permalink with this on the view file page (this has no repo or commit)
![07:16:12](https://github.com/user-attachments/assets/f8f10868-e965-45a2-a955-73ed8eec066d)